### ISSUE TITLE
Cry for HELP to all (Port setup)

### ISSUE THREAD

**Issue by aaluck1** [bklavet/Echo-skill-to-control-Directv#61]
First off.....I have never programmed anything and can't really believe I have gotten this far. So forgive my lay terms for technical items.

Here is what I have....  The skill works for me "internally".  I can pause the receiver from my home computer, via the command B money provided me. So the skill is set up correctly.  However, when I run the function test or speak to Alexa it says it cannot find the receiver.  So we (me and B money, B money mainly since I'm ignorant) have determined that the problem is the port settings not allowing the skill to pass the router and go to the receiver.

I have a Mac Mini, and Airport Extreme as my router.  WOW is my provider and there are no blocks set up according to them that would prevent the signal from entering. I have messed around with the ports and thought maybe I have done it right but I am not a computer guy.

Has anyone set this up on an Airport successfully?  If so, could you PLEASE post screen shots of how you did it, step by step, OR at least the DHCP Reservations (if used) and the Port Settings.  Also have you checked the "Enable Access Control" and set up anything in the "Network Options".

Additional information that may be helpful is, I can access my DVR from the Directv app, but could do that before.  This leads me to believe there is an "opening" that allows outside access, but the Alexa skill obviously cannot use or does not use this same access point.

**Comment by aaluck1** [comment]
Okay. After I posted this I reread #12 and found this test port site and it is open.  So any ideas as to what it could be at this point.

![screen shot 2018-01-05 at 6 30 23 pm](https://user-images.githubusercontent.com/34351600/34634170-057545fe-f247-11e7-9e23-915b582f9c01.png)

**Comment by bklavet** [comment]
that ip looks odd.
what to double check with this
https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=2&cad=rja&uact=8&ved=0ahUKEwjhv8XojsLYAhVB4YMKHYZTDK8QFgguMAE&url=https%3A%2F%2Fnetworkappers.com%2Ftools%2Fopen-port-checker&usg=AOvVaw0q4gdLX-tqJKUP-XibfmaX

**Comment by vmweaver** [comment]
That IP is owned by Wide Open West, so that makes sense. I would recommend
obfuscating that in your screen shot, though.  It indeed seems to be open
and up, so it shouldn't be connectivity to the DTV box at this point.

**Comment by vmweaver** [comment]
Verify that the line in the Index file:
var local_ip = 'XXX.XXX.XXX.XXX'; 

is NOT your "local_ip", but rather the 216.XXX.XXX.XXX address

Should probably look to update that variable name to reduce confusion, too. ;-)

**Comment by bklavet** [comment]
I googled it and it looked like it was out of italy.... that said, I was able to get the locations from it so Forwarding looks good.  Happy new year mawrew19!
aaluck1 want to send me a comment on the youtube channel with your email.  I would be interested in seeing your zip file you are uploading to lambda.

"Should probably look to update that variable name to reduce confusion, too. ;-)"
yeah, your right.  Thanks my friend!
